src/lib/incoming_message.ts

```diff
@@ -7,7 +9,14 @@ interface IncomingCommandStartListening extends IncomingCommandBase {
   command: "start_listening";
 }
 
+interface IncomingCommandUpdateLogConfig extends IncomingCommandBase {
+  messageId: string;
+  command: "update_log_config";
+  config: DeepPartial<LogConfig>;
```

Comment:
Why is this `DeepPartial` and not `Partial` ? Can partial transport configs be defined ?

Reply:
I used the type that is expected upstream: https://github.com/zwave-js/node-zwave-js/blob/master/packages/zwave-js/src/lib/driver/Driver.ts#L590

Will `Partial` work instead?

I don't know anything about the `Transport` part of the config but it seems that is mainly for internal use, e.g. it shouldn't be updated externally (https://github.com/zwave-js/node-zwave-js/blob/master/packages/core/src/log/shared.ts#L129)

Reply:
I looked into `DeepPartial` and `Partial` and since we have decided not to support updating transports, I don't think `DeepPartial` is needed